code/visualize.py

```diff
@@ -14,14 +17,17 @@ def create_single_file_chart(filename):
     plot.set_xticklabels(x_values)
     plt.title(get_word_from_path(filename))
     plt.xlabel('Months')
-    plt.ylabel('Word count')
+    plt.ylabel(y_axis_count_parameter + ' count')
 
     plt.show()
 
 
 def get_word_from_path(path):
     filename = os.path.basename(path)
-    return ''.join(list(reversed(filename.split('_')[0])))
+    if any("\u0590" <= c <= "\u05EA" for c in filename.split('_')[0]):
```

Comment:
Cool! If you don't mind, extract these to consts so we could use them in other places too.